Readme.md

```diff
@@ -1,3 +1,8 @@
 # Taxi App
 
 A cross platform UI template to build a Taxi Booking App that provides all the necessary screens along with NativeBase and React Navigation.
+
+### Demo
+This is an design app, so APIs are not connected. You can add dummy data while checking it out.
+
```

Comment:
Add a line here:
"For instance, you can type any number on the OTP and mobile number field for going to next screen "